### DISCUSSION TITLE
[Feature] Ability to test machine learning server URL reachability

### DISCUSSION THREAD

**Discussion by TheDidek** [immich-app/immich#10685] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Ref to: https://immich.app/docs/guides/remote-machine-learning

Immich settings interface does not let user check if the typed URL is actually reachable, making it hard to see mistake when using more complex network setups.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by Kieranwest** [comment]
This would be great.
